common/tools/eslint-plugin-azure-sdk/src/configs/azure-sdk-base.ts

```diff
@@ -5,7 +5,7 @@ export default {
   parser: "@typescript-eslint/parser",
   parserOptions: {
     sourceType: "module",
-    extraFileExtensions: [".json", ".javascript"],
```

Comment:
IIRC ".javascript" is the extension for the code snippets in markdown files from `"plugin:markdown/recommended-legacy"` but without a on disk test fixture the rule probably doesn't work anyway.

https://github.com/Azure/azure-sdk-for-js/blob/e905eaf35a8b80abef6265a10921ef824b6849ee/common/tools/eslint-plugin-azure-sdk/src/configs/azure-sdk-base.ts#L143-L144

Reply:
Can take a look again when we upgrade to 8, but the examples in their README makes it seem like we can use `js` for the block language rather than javascript: https://www.npmjs.com/package/eslint-plugin-markdown